build.fsx

```diff
@@ -94,14 +94,20 @@ let setParams defaults = {
             ]
     }
 
+let Exec command args =
+    let result = Shell.Exec(command, args)
+    if result <> 0 then failwithf "%s exited with error %d" command result
+
 Target "BuildApp" (fun _ ->
     build setParams "./Octokit.sln"
         |> DoNothing
 )
 
-Target "BuildXSApp" (fun _ ->
-    build setParams "./Octokit-XamarinStudio.sln"
-        |> DoNothing
+Target "BuildMono" (fun _ ->
+    // xbuild does not support msbuild  tools version 14.0 and that is the reason
+    // for using the xbuild command directly instead of using msbuild
+    Exec "xbuild" "./Octokit-Mono.sln /t:Build /tv:12.0 /v:m  /p:RestorePackages='False' /p:Configuration='Release' /logger:Fake.MsBuildLogger+ErrorLogger,'../octokit.net/tools/FAKE.Core/tools/FakeLib.dll'"
```

Comment:
Interesting, thanks. Looks like the default `xbuild` script invokes xbuild 12.0, which is probably why this happens. Not sure if that's intended, I'll take a closer look.